src/containers/Login/Login.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+import './Login.scss';
+import { connect } from 'react-redux';
+import { loginUser } from '../../actions';
+
+class Login extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      username: '',
+      password: ''
+    };
+
+    this.handleUsernameOnChange = this.handleUsernameOnChange.bind(this);
+    this.handlePasswordOnChange = this.handlePasswordOnChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  };
+
+  componentWillMount() {
+    document.addEventListener('mousedown', this.handleClick, false);
+    document.querySelector('.columnContainer').classList.add('blur');
```

Comment:
commented
Instead of using document, try to directly add the class in the render function using className and the ternary operator